see https://code.videolan.org/videolan/dav1d/-/issues/423
we pass -flto in cflags as well as b_lto, so the check breaks
--
From 007d789a9e05ef4265dbfb5366903bcb1a1077fb Mon Sep 17 00:00:00 2001
From: q66 <q66@chimera-linux.org>
Date: Fri, 2 Sep 2022 21:01:53 +0000
Subject: [PATCH] disable HAVE_AS_FUNC check

This passes but the asm code does not actually compile with it.
---
 meson.build | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/meson.build b/meson.build
index a312ae0..763c9c6 100644
--- a/meson.build
+++ b/meson.build
@@ -442,7 +442,7 @@
     endif
 endif
 
-cdata.set10('HAVE_AS_FUNC', have_as_func)
+cdata.set10('HAVE_AS_FUNC', false)
 cdata.set10('HAVE_AS_ARCH_DIRECTIVE', have_as_arch)
 foreach name, _ : aarch64_extensions
     cdata.set10('HAVE_AS_ARCHEXT_' + name.to_upper() + '_DIRECTIVE', name in supported_aarch64_archexts)
